server/pages/index.js

```diff
@@ -38,6 +40,12 @@ const staticFiles = [
 
 const sections = sectionsModule.get();
 
+function getInitialServerState( serializedServerState ) {
+	// Bootstrapped state from a server-render
+	const serverState = reducer( serializedServerState, { type: DESERIALIZE } );
+	return pick( serverState, Object.keys( serializedServerState ) );
```

Comment:
> What is `pick` effectively weeding out?

Currently, we only persist stuff on the server that we need for SSR (`document-head`, `themes`, and `ui` subtrees). We omit anything else because we don't need it, and it might unnecessarily bloat our cache.